app/modules/DataBase/connections.ts

```diff
@@ -0,0 +1,15 @@
+let mongoose = require("mongoose");
+import config from "../../config";
+
+const server = "127.0.0.1:27017";
```

Comment:
```suggestion
const server = "127.0.0.1:27017";
``` remove this magic strings